shaw/roster.json

```diff
@@ -26,6 +26,10 @@
         "name": "Matthew Lo",
         "github": "matthewlohl"
       },
+       {
+        "name": "Shaeeb Ali",
+        "github": "shaeebali"
+      },  
```

Comment:
Please remove the trailing comma and make sure that your curly brackets are both aligned :)